GlobalViolence/R/DataPrep_GBD.R

```diff
@@ -48,26 +50,35 @@ GBD$metric <- recvec[GBD$metric]
 GBD <- GBD[, .(val = sum(val), upper = sum(upper) ,lower = sum(lower)),
 		by = .(metric, location, year, sex, cause, age)];gc()
 
-# ISO Codes for mapping. This should be earlier in processing, move at some point.
-ISO           <- read.csv(here("GlobalViolence","Data","Inputs","GBD","GBD_ISO3.csv"),
-                          stringsAsFactors = FALSE)
-recvec        <- ISO[, 2]
-names(recvec) <- ISO[, 1]
-GBD$ISO3      <- recvec[GBD$location]
+
+
+#For some reason this is not working, at least for me. Taiwan ISO code (TWN) disappears, generating missing later.
+# these are the first observations. The rest is fine though. 
+# Instead of doing all of this maybe we could just apply country code now. See suggestion below
+# Using countrycode here instead of code lines above:
+
+GBD$ISO3<-countrycode(GBD$location, origin="country.name", destination="iso3c")
```

Comment:
yeah, this is much better. I did the `recvec` thing for years and have now started using `*_join()` operations for the same thing, as it's less laborious. But this bespoke function should be better, word.

Reply:
we could also just use _join later as well in the map codes. I will use it anyway to left_join the ISO codes from this dataset to the shapefile via ISO codes. We can leave it in the open when we do it.